Wrapping/Generators/Python/PyBase/pyBase.i

```diff
@@ -580,19 +580,27 @@ str = str
     %typemap(in) type& (type itks) {
         if ((SWIG_ConvertPtr($input,(void **)(&$1),$1_descriptor, 0)) == -1) {
             PyErr_Clear();
-            itks = type( PyObject_Length($input) );
-            for (unsigned int i =0; i < itks.Size(); i++) {
-                PyObject *o = PySequence_GetItem($input,i);
-                if (PyInt_Check(o)) {
-                    itks[i] = (value_type)PyInt_AsLong(o);
-                } else if (PyFloat_Check(o)) {
-                    itks[i] = (value_type)PyFloat_AsDouble(o);
-                } else {
-                    PyErr_SetString(PyExc_ValueError,"Expecting a sequence of int or float");
-                    return NULL;
+            if (PySequence_Check($input)) {
+                itks = type( PyObject_Length($input) );
+                for (unsigned int i =0; i < itks.Size(); i++) {
+                    PyObject *o = PySequence_GetItem($input,i);
```

Comment:
According to:

https://docs.python.org/3/c-api/sequence.html#c.PySequence_GetItem

`PySequence_GetItem` returns a new reference -- if I recall correctly, the CPython decrement reference macros need to be called after you are finished using the object and in the error conditions to avoid a memory leak.

https://docs.python.org/2/c-api/refcounting.html